internal/store/store.go

```diff
@@ -268,6 +268,15 @@ func (store *Store) GetDel(k string, opts ...DelOption) *object.Obj {
 	return v
 }
 
+// GetUnixTimeExpiry returns the expiry time for an object.
+func (store *Store) GetUnixTimeExpiry(obj *object.Obj) (uint64, bool) {
```

Comment:
we have a function already created for this: GetExpiry() from expiry.go. Please try to reuse that function as all other evals use this.